openbb_terminal/stocks/options/options_controller.py

```diff
@@ -1222,6 +1224,71 @@ def call_greeks(self, other_args: List[str]):
                     show_all=ns_parser.all,
                 )
 
+    @log_start_end(log=logger)
+    def call_var(self, other_args: List[str]):
+        """Process VaR command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="var",
+            description="The value at risk for a given option at expiry, thus mostly suitable for european options.",
+        )
+        parser.add_argument(
+            "-s",
+            "--strike",
+            dest="strike",
+            type=float,
+            help="The strike of the option",
+        )
+        parser.add_argument(
+            "-sim",
+            "--simulations",
+            dest="simulations",
+            type=float,
+            default=100,
+            help="The number of simulations",
+        )
+        parser.add_argument(
+            "-per",
+            "--percentile",
```

Comment:
super nitpick but we want to keep any arguments with "-" to one letter for unix convention